dokka-integration-tests/gradle/src/testExternalProjectKotlinxCoroutines/kotlin/CoroutinesGradleIntegrationTest.kt

```diff
@@ -38,12 +38,12 @@ class CoroutinesGradleIntegrationTest : AbstractGradleIntegrationTest(), TestOut
     override val projectOutputLocation: File by lazy { File(projectDir, "build/dokka/htmlMultiModule") }
 
     @BeforeTest
-    fun prepareProjectFiles() {
-        val templateProjectDir = File("projects", "coroutines/kotlinx-coroutines")
-        templateProjectDir.listFiles().orEmpty()
-            .forEach { topLevelFile -> topLevelFile.copyRecursively(File(projectDir, topLevelFile.name)) }
-
-        copyAndApplyGitDiff(File("projects", "coroutines/coroutines.diff"))
+    override fun beforeEachTest() {
```

Comment:
Is there a reason why serialisation and coroutines setup differs?
In serialisation we don't override this method, but introduce another one?

Reply:
Good spot! I've made both setup tasks work the same